skaffold.yaml

```diff
@@ -152,4 +153,14 @@ deploy:
           enrollments.creds[1].options: "--id.type peer"
           enrollments.csrHost: network-org-2-peer-1-hlf-peer.org-2
           toolbox.enabled: true
-
+          hooks.serviceAccount.name: substra-delete-hook
+    flags:
+      install: ["--create-namespace"]
+  # We create the serviceAccounts with the kubectl deployer to ensure the serviceAccounts will stay present for the Helm deletion hooks
+  # By design Skaffold create and delete ressources in a fixed order (Helm > kubectl > kustomize)
+  # Ref: https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/runner/new.go#L194-L215 (func getDeployer)
```

Comment:
This file has been updated since, and the line numbers don't match anymore. Use a commit-hash specific link (instead of master)?